pkgs/development/libraries/qxt/default.nix

```diff
@@ -0,0 +1,27 @@
+{ stdenv, which, coreutils, fetchzip, qt4 }:
+
+stdenv.mkDerivation rec {
+  name = "qxt-${version}";
+  version = "0.6.2";
+  
+  src = fetchzip {
+    url = "http://dev.libqxt.org/libqxt/get/v${version}.tar.gz";
+    sha256 = "0zmqfn0h8cpky7wgaaxlfh0l89r9r0isi87587kaicyap7a6kxwz";
+  };
+
+  buildInputs = [ qt4 which ];
+
+  patchPhase = ''
+    patchShebangs configure
+    substituteInPlace configure --replace "/bin/pwd" "${coreutils}/bin/pwd"
+  '';
+
+  prefixKey = "-prefix ";
+
+  meta = {
+    homepage = http://libqxt.org;
+    description = "An extension library for Qt providing a suite of cross-platform utility classes to add functionality not readily available in the Qt toolkit by Qt Development Frameworks, Nokia.";
```

Comment:
Sounds fine to me. Will do it when I get a chance (working on a different package right now), but what about the short description?